scripts/scrape-metrics.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright (c) 2024 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euf -o pipefail
+
+print_help() {
+  echo "Usage: $0 [-c component] [-o output_dir]"
+  echo "  -c: Jaeger component (agent, collector, query, ingester, all-in-one)"
+  echo "  -o: Output directory for metrics files"
+  exit 1
+}
+
+# Default values
+component=""
+output_dir="./metrics"
+
+# Jaeger component metrics ports
+declare -A COMPONENT_PORTS=(
+    ["agent"]=14271
+    ["collector"]=14269
+    ["query"]=16687
+    ["ingester"]=14270
+    ["all-in-one"]=14269
```

Comment:
irrelevant, these are all v1 components, we don't run e2e tests with binaries of those components so there's no place to scrape metrics from. Only v2 binary is used as binary in e2e tests